.env.example

```
@@ -12,10 +18,11 @@ REDIS_PASSWORD=
 JWT_SECRET=
 JWT_ACCESS_TOKEN_EXPIRATION_MINUTES=30
 JWT_REFRESH_TOKEN_EXPIRATION_DAYS=14
-JWT_REFRESH_TOKEN_COOKIE_SECURE=true
+# HTTP 환경에서는 false, HTTPS 적용 후에는 true로 변경합니다.
+JWT_REFRESH_TOKEN_COOKIE_SECURE=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The JWT_REFRESH_TOKEN_COOKIE_SECURE key should go before the JWT_REFRESH_TOKEN_EXPIRATION_DAYS key

(UnorderedKey)